packages/next/src/routes/rest/routeError.ts

```diff
@@ -47,7 +47,10 @@ export const routeError = async ({
 
   let status = err.status || httpStatus.INTERNAL_SERVER_ERROR
 
-  logger.error(err.stack)
+  const level = payload.config.loggingLevels[err.name] ?? 'error'
+  if (level) {
+    logger[level](level === 'info' ? err.message : err.stack)
```

Comment:
The pino error logger should accept an object w/ `err` property. This way it will map the message and stack properly. The usage is like this:

`logger.error({ err })`